### ISSUE TITLE
bug: App not working on modern Huawei devices

### ISSUE THREAD

**Issue by sabereen** [ionic-team/capacitor#6258]
### Capacitor Version
```
  @capacitor/cli: 4.6.2
  @capacitor/core: 4.6.2
  @capacitor/android: 4.6.2
  @capacitor/ios: 4.6.2

Installed Dependencies:

  @capacitor/ios: not installed
  @capacitor/core: 4.6.2
  @capacitor/cli: 4.6.2
  @capacitor/android: 4.6.2
```

### Platform(s)
android

### Current Behavior
Huawei has its own webview and not using google`s one.
Its versions is not same as chrome!
Capacitor detects it as legacy webview (<60) even in modern webviews and redirects the user to errorPath.

### Expected Behavior
App should working in modern Huawei devices!

**Comment by AntonNB** [comment]
Pls, fix bug

**Comment by AntonNB** [comment]
Same issue on Honor devices. Apparently, when there was no GMS out of the box, problems arise.

**Comment by maningame** [comment]
Have the same issue, so I did some research. The problem is that the Huawei webview have its own versioning, which is different from Android system webview. Current is 12.1.3.371 or something like this.

Capacitor check version by WebView.getCurrentWebViewPackage() and its return 12.x.x.x (package version, not a chrome version inside it). I made quick workaround by adding the folllowing code to Bridge.java > isMinimumWebViewInstalled() after PackageInfo info = WebView.getCurrentWebViewPackage():

```
if (info.packageName.equals("com.huawei.webview")) {
    return true;
}
```

Its allow to bypass the check, but obviously the minWebViewVersion became useless in that case because we cant do version matching between huawei webview and android webview. To solve this issue I do additional check in my app. Hope this was helpfull.
